diff --git a/CMakeLists.txt b/CMakeLists.txt
index 136ed42..4d468f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -95,6 +95,13 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   )
 endif()
 
+set_target_properties(
+  AtomicParsley PROPERTIES
+      CXX_STANDARD 11
+      CXX_STANDARD_REQUIRED YES
+      CXX_EXTENSIONS NO
+)
+
 if (ASAN)
   set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
   set (CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_LINKER_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
